src/style.css

```
@@ -0,0 +1,77 @@
+@font-face {
+  font-family: "Avenir-Book";
+  font-style: normal;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  font-weight: bold;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.bar-form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.bar {
+  width: 100%;
+  font-family: Avenir-Book, sans-serif;
+  color: #3d4e61;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.bar-big {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.bar-small {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.icon-big {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  display: block;
+  width: 19px;
+  height: 19px;
```

Зроби відступи між різними групами правил, тоді твій код буде простіше читати і підтримувати.
```suggestion
  position: absolute;
  top: 25px;
  left: 26px;
  
  display: block;
  width: 19px;
  height: 19px;
```

---

done